client/src/components/main/splash.scss

```diff
@@ -69,19 +58,6 @@ $background-image: 'https://roastia.s3.us-east-2.amazonaws.com/roastia-home.jpeg
   margin-bottom: 25px;
 }
 
-.splash-icons {
-  @media only screen and (max-width: 1200px) {
```

Comment:
Same comment here about keeping all of these above the fold (was a big pain in the butt to figure out originally for me.)

Reply:
Like above, I just moved the stylesheet into the react component 
https://github.com/hegelocampus/Roastia/blob/152b2f4a1ae2feea3a62725850d288bd2aa4bb6b/client/src/components/util/icons.js#L11-L23